### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by Minseung-Kim** [anicolson/DeepXi#48]
Hello, Thank you for sharing the good projects. 
I have some questions.

1. for the demand_voice_bank dataset, the described best model (in terms of PESQ) is resnet-1.1c (ResNetV2). Then does ResNetV3(which has a very slight difference fromResNetV2) give better/worse performance compared to the ResNetV2? Have you ever tried to compare?

2. for the demand_voice_bank dataset, does the validation set (if it is used) is fixed? and
As far as I understand, each utterance in the training set is mixed with random SNR (in the range of [0 5 10 15]) with fixed clean/noise pair) for each epoch. Is it correct..?

3. I can get the mu/sigma for the CDF mapping function in the data/stats.mat. Is it OK to use these parameters for the demand_voice_bank dataset? OR do I need to calculate the new mu/sigma parameter set?

4. In deepxi/gain, the function 'deepmmse' returns (1/1+xi) + (xi/(gamma(1+xi))).
But.. is it right that (1/1+xi)^2 + (xi/(gamma(1+xi)))?

Thank you.

**Comment by anicolson** [comment]
Hi Minseung-Kim,

For 1) ResNetV3 is just an amendment to ResNetV2 (a very minor change). From memory, there is a negligible difference between them. You should get the same performance from either. 

For 2) The validation set is fixed. For the training set, everything is correct except that the noise recording used to corrupt the clean speech is randomly selected.

For 3) It is OK, however, you may get a little bit more performance with statistics tailored for the demand_voice_bank dataset. This is something that I have not tested.

For 4) Yes, you are right (@yunzqq can double-check). I will update the code in the meantime.

Please let us know if you need more help.

**Comment by Minseung-Kim** [comment]
Thank you for your quick response!
